index.rst

```diff
@@ -260,92 +291,98 @@ If working with an individual CSC, which as an operator would be a rare occurren
 
 .. TODO: Add example on how to launch script from LOVE interface
 
+.. _section-configuration-creating-a-new:
 
-.. _section-configuration-interation_changing_default:
+Modifying or Creating a New Configuration
+=========================================
 
-Changing the default configuration
-----------------------------------
+The process to derive new configuration parameters varies considerably from component to component.
+In some cases, the configuration is simple enough that a change may involve simply replacing an IP or hostname value, a routine filter swap on an instrument or updating the limits to an axis range due to some evolving condition.
+On the other hand, deriving new parameters may involve generating complex LUTs that may require on sky observations and detailed data analysis.
 
-Changing the default configuration is a more involved endeavor because it entails making a change to the contents of the configuration repository.
-Because the repo is under version control, the appropriate steps must be taken.
-For this example, let's assume we want to change the default configuration in the ATAOS, which is found in the `ATAOS directory of the ts_config_attcs repo <https://github.com/lsst-ts/ts_config_attcs/tree/develop/ATAOS>`__.
+Following is a detail of each step of the process to generate a new configuration and update it for CSCs written in salobj.
+For other components, see the exception section below.
 
 
-#.  Create a JIRA ticket in where the title/description note the change being made.
-    Let's assume it creates ticket DM-12345.
+#.  Create a Jira ticket to track the work being done (e.g. DM-12345).
+    If details or discussions are needed they can done using the Jira tickets itself.
 
```

Comment:
I think this prompt bellow is a bit lost without the item that tells users they need to clone and create a new branch.